package.json

```diff
@@ -74,7 +74,7 @@
     "@hmcts/nodejs-healthcheck": "^1.7.0",
     "@hmcts/properties-volume": "^0.0.9",
     "@hmcts/rpx-xui-common-lib": "0.2.5",
-    "@hmcts/rpx-xui-node-lib": "2.25.0",
+    "@hmcts/rpx-xui-node-lib": "2.25.2",
```

Comment:
What happened to `2.25.1`? 🤔

Reply:
2.25.1 does not have logging. So had to create one additional version